packages/modules/src/modules/header/HeaderWrapper.tsx

```diff
@@ -48,6 +50,23 @@ export const headerWrapper = (Header: React.FC<HeaderRenderProps>): React.FC<Hea
             secondaryCta,
         };
 
+        const mobilePrimaryCta = mobileContent?.primaryCta
+            ? buildEnrichedCta(mobileContent?.primaryCta)
+            : primaryCta;
+
+        const mobileSecondaryCta = mobileContent?.secondaryCta
+            ? buildEnrichedCta(mobileContent?.secondaryCta)
+            : secondaryCta;
```

Comment:
Just a tiny point here - I guess we don't need the optional chaining after we start the ternary i.e just

```ts
buildEnrichedCta(mobileContent.primaryCta)
```